cmd/algokey/keyreg.go

```diff
@@ -82,7 +82,7 @@ func init() {
 	keyregCmd.Flags().BoolVar(&params.offline, "offline", false, "set to bring an account offline")
 	keyregCmd.Flags().StringVarP(&params.txFile, "outputFile", "o", "", fmt.Sprintf("write signed transaction to this file, or '%s' to write to stdout", stdoutFilenameValue))
 	keyregCmd.Flags().StringVar(&params.partkeyFile, "keyfile", "", "participation keys to register, file is opened to fetch metadata for the transaction; only specify when bringing an account online to vote in Algorand consensus")
-	keyregCmd.Flags().StringVar(&params.addr, "account", "", "account address to bring offline; only specify when taking an account offline from voting in Algorand consensus")
+	keyregCmd.Flags().StringVar(&params.addr, "account", "", "account to bring offline; only specify when taking an account offline from voting in Algorand consensus")
```

Comment:
I think that the word "address" should remain since the provided parameter *is* the address.
Also, you're referring to the operation as both "bring offline" and "taking offline". I think it would be cleaner to align with one of these. ( I *think* that we're using the term "take offline" more frequently. But I could be wrong here ).

Reply:
I'll add "address" back in. This is the phrasing Noah suggested, I don't feel very strongly about it. It looks like goal doesn't use either of these and is using language like `set to bring an account offline`.